dev/builddeps-veloxbe.sh

```diff
@@ -16,8 +16,6 @@ ENABLE_HBM=OFF
 BUILD_PROTOBUF=ON
 ENABLE_S3=OFF
 ENABLE_HDFS=OFF
-BUILD_ARROW_FROM_SOURCE=ON
-BUILD_VELOX_FROM_SOURCE=ON
```

Comment:
Why do we need to remove the two configs? I find they are useful sometimes if we don't want to compile one of them from source.

Reply:
From my point of view, hongze make these two configs replaced by enable_ep_cache.

Reply:
Looks previously these two options only control the fetch of source code. LGTM.